services/ux-backend/handlers/bucket/handler.go

```diff
@@ -0,0 +1,97 @@
+package bucket
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/red-hat-storage/ocs-operator/v4/services/ux-backend/handlers"
+	nbv1alpha1 "github.com/noobaa/noobaa-operator/v5/pkg/apis/noobaa/v1alpha1"
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	CreationMethodOBC = "obc"
+	CreationMethodS3  = "s3"
```

Comment:
are these being exported?

Reply:
no...
should I use camelCase format for the variable name ??

Reply:
updated...